tests/test_domain_models.py

```
@@ -0,0 +1,121 @@
+import pytest
+from decimal import Decimal
+from datetime import datetime
+from src.domain.models import (
+    Coordinates, Location, Sensor, Measurement,
+    ParameterType, MeasurementUnit
+)
+
+
+class TestCoordinates:
+    def test_valid_coordinates(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_valid_coordinates`

Add return type annotation: `None`

(ANN201)